jsonschema-hyperschema.xml

```diff
@@ -746,14 +780,15 @@
 
             <section title="rel" anchor="rel">
                 <t>
-                    The value of the "rel" property indicates the name of the relation to the target
+                    The value of the "rel" property indicates the name of the relation type from
+                    the link's context resource to its target
                     resource. The value MUST be a registered link relation from the
                     <xref target="RFC5988">IANA Link Relation Type Registry established in RFC 5988</xref>,
                     or a normalized URI following the <xref target="RFC3986">URI production of RFC 3986</xref>.
                 </t>
 
                 <t>
-                    The relation to the target is interpreted as from the instance that the schema
+                    The link context is interpreted as from the instance that the schema
```

Comment:
I'm not sure the "from" is still needed. I'd read it better as `The link context is interpreted as the instance that the schema (or sub-schema) applies to, [...]`.